junebug/tests/test_logging_service.py

```diff
@@ -207,6 +211,22 @@ def test_start_stop(self):
         self.assertFalse(self.service.registered())
         self.assertEqual(self.service.logfile.closed_count, 1)
 
+    @inlineCallbacks
+    def test_dir_create(self):
+        '''If log directory already exists, make sure it is not recreated.'''
+        if not os.path.exists(self.service.path):
+            os.makedirs(self.service.path, 0777)
+        self.stat1 = os.stat(self.service.path)
```

Comment:
We can just use a local variable here since there's no need to access `stat1` outside of this method. (so `stat1 = ...` rather than `self.stat1 = ...`). The same goes for `stat2`.